src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -251,4 +251,39 @@ public Optional<NetworkModificationResult> duplicateModifications(UUID targetGro
         }
         return Optional.empty();
     }
+
+    public UUID createModificationInGroup(@NonNull ModificationInfos modificationsInfos) {
+        UUID groupUuid = UUID.randomUUID();
+        networkModificationRepository.saveModifications(groupUuid, List.of(modificationsInfos.toEntity()));
+        return groupUuid;
+    }
+
+    @Transactional
+    public Optional<NetworkModificationResult> duplicateModificationsInGroup(UUID targetGroupUuid,
```

Comment:
We already have this feature when copying nodes full of network-modifications don't we ?

Reply:
I have refactored the code to use a common private method